apis/hive/v1/metricsConfig/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package metricsConfig contains API Schema definitions for configurations specific to metrics controller.
+// +k8s:deepcopy-gen=package,register
+package metricsConfig
```

Comment:
Still v1 scoped, let me know if you'd like me to move these files elsewhere

Reply:
No, I think this works.

I'm not sure if there's a problem using camelCase for the package name though...

Reply:
So golang package rules suggest all lowercase - fixed it